pandas/_libs/skiplist.pyx

```diff
@@ -5,144 +5,3 @@
 # Link: http://code.activestate.com/recipes/576930/
 
 # Cython version: Wes McKinney
```

Comment:
Is there a reason that we need this file (now without any code) ?

Reply:
No, we'd need to update setup.py